uniffi/src/interface/mod.rs

```diff
@@ -548,13 +564,21 @@ impl Object {
         self.methods.iter().collect()
     }
 
+    pub fn reserved_object_free(&self) -> &Method {
+        &self.object_free
+    }
+
     fn derive_ffi_funcs(&mut self, ci_prefix: &str) -> Result<()> {
         for cons in self.constructors.iter_mut() {
             cons.derive_ffi_func(ci_prefix, &self.name)?
         }
         for meth in self.methods.iter_mut() {
             meth.derive_ffi_func(ci_prefix, &self.name)?
         }
+        let reserved_prefix = format!("_{}", ci_prefix);
+        self.object_free
+            .derive_ffi_func(&reserved_prefix, &self.name)?;
+
```

Comment:
Adding a `_` prefix makes it much harder to generate colliding method names and top level function names.